jupyterhub_traefik_proxy/traefik_utils.py

```diff
@@ -22,14 +22,34 @@ def validate(self, obj, value):
 
 
 def generate_rule(routespec):
+    # validate assumption that routespecs always end with '/'
```

Comment:
Can we add a tiny comment here about what this function does and/or link to https://doc.traefik.io/traefik/routing/routers/#rule?

Reply:
Amazing! Thank you!